content/security-settings/contents.lr

```diff
@@ -0,0 +1,56 @@
+_model: topic
+---
+title: Security Settings
+---
+description: Configuring Tor Browser for security and usability
+---
+seo_slug: security-settings
+---
+key: 8
+---
+body:
+By default, Tor Browser protects your security by encrypting your browsing data.
+You can further increase your security by choosing to disable certain web features that can be used to attack your security and anonymity.
+You can do this by increasing Tor Browser's Security Settings in the shield menu.
+Increasing Tor Browser's security level will stop some web pages from functioning properly, so you should weigh your security needs against the degree of usability you require.
```

Comment:
Maybe we can link here to some of the documents we have to help people assess their threat model?